tests/publiccloud/create_aistack_env.pm

```diff
@@ -0,0 +1,276 @@
+    # docker activation section
+    process_reboot(trigger => 1);
+    systemctl("enable docker");
+    systemctl("start docker");
+    systemctl("status docker");
+    # rke2 activation section
+    script_run("curl -sSL $rke2_url -o ./install_rke2.sh && chmod 775 ./install_rke2.sh");
+    script_run("sh ./install_rke2.sh");
+    script_run("echo 'export PATH=\$PATH:/opt/rke2/bin' >> ~/.bashrc");
+    systemctl("enable rke2-server.service");
+    systemctl("start rke2-server.service");
+    systemctl("status rke2-server.service");
+    script_run("rke2 --version");
+    # helm activation section
+    script_run("curl -sSL $helm_url -o ./install_helm.sh && chmod 775 ./install_helm.sh");
+    script_run("sh ./install_helm.sh");
+    script_run("helm version");
+    # kubectl activation section
+    script_run("curl -sSL $kubectl_url -o ./kubectl && chmod +x ./kubectl");
+    script_run("sudo mv ./kubectl /usr/local/bin/");
+    script_run("kubectl version --client");
+}
+
+sub install_dependency_components {
+    my ($instance) = @_;
+    my $cert_repo = get_var('HELM_CERTS');
+    my $ingress_repo = get_var('HELM_INGRESS');
+    my $ing_ver = get_var('ING_VERSION');
+
+    # Add Ingress Controller to open-webui endpoint
+    assert_script_run("helm repo add $ingress_repo");
+    assert_script_run("helm repo update");
+    assert_script_run("helm upgrade --install ingress-nginx ingress-nginx/ingress-nginx --namespace ingress-nginx --set controller.service.type=ClusterIP --version $ing_ver --create-namespace", timeout => 120);
+
+    # Add cert-manager repo,install
+    assert_script_run("helm repo add $cert_repo");
+    assert_script_run("helm repo update");
+    assert_script_run("helm install cert-manager jetstack/cert-manager --namespace cert-manager --create-namespace --version v1.15.2 --set crds.enabled=true", timeout => 120);
+}
+
+sub config_kubectl {
+    my ($instance) = @_;
+    # config kubectl
+    record_info('CONFIG kubectl');
+    assert_script_run("mkdir -p ~/.kube");
+    assert_script_run("sudo cp /etc/rancher/rke2/rke2.yaml ~/.kube/config");
+    assert_script_run("kubectl config get-contexts");
+    assert_script_run("kubectl config use-context default");
+    assert_script_run("kubectl config view");
+}
+
+sub install_aistack_chart {
+    my ($instance, $ai_chart_repo, $namespace, $vf_name) = @_;
+    my $SECRET_application_collection = get_var('_SECRET_DOCKER');
+    my $cert_repo = get_var('HELM_CERTS');
+    my $ingress_repo = get_var('HELM_INGRESS');
+    my $docker_user_name = 'yarunachalam@suse.com';
+    #my $docker_user_name = get_var('USER_DOCKER');
+    my $repo_url = get_var('HELM_CHARTS');
+    my $git_token = get_var('_SECRET_SSH');
+    my $ing_ver = get_var('ING_VERSION');
+    my $local_storage_name = 'local_path_storage.yaml';
+    my $local_path_url = 'https://raw.githubusercontent.com/os-autoinst/os-autoinst-distri-opensuse/54e38954acd015490d85e7614c00c54db46ed9f7/data/aistack/local_path_storage.yaml';
```

Comment:
```suggestion
    my $local_path_url = data_url('local_path_storage.yaml');
```
?

Reply:
@yarunachalam this isn't still resolved, but I'm curious now, how are you running your verification jobs, that this seems needed?

Reply:
this was fixed on my last commit
I did not do verification run after this change.. this file is part of this PR.